src/main/resources/static/css/home-page-style.css

```
@@ -0,0 +1,81 @@
+body{
+    background-color: #42BFDD;
+
+}
+
+p{
+    font-size: 24px;
+  }
+
+h2{
+    font-size: 48px;
+  }
+
+.button {
+    background-color: #1D77C3; 
+    border: none;
+    color: white;
+    padding: 15px 50px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    width: 85%;
+    gap: min(2px, 1%);
+  
+  }
+  .button:hover {
+    cursor: pointer;
+    background-color: lightslategray;
+  }
+
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  grid-column-gap: 10px;
+  grid-row-gap: 10px;
+  height: 100vh; 
+  width: 100%;
+}
+  
+  .grid-item-2-img{
+  width: 100%;
+  height: auto;
```

Consistent indenting
```suggestion
    width: 100%;
    height: auto;
```